week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,17 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
+  if (
+    typeof num === "number" &&
+    num === Math.round(num) &&
+    excludedNums.includes(num) === false
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
Good :+1: . Remember if you have code `if (something) { return true } else { return false }` then that is the same as `return something`. So this could be:

```
return (typeof num === "number" && num === Math.round(num) && !excludedNums.includes(num))
```
I also used the "not" operator instead of ` === false`.